src/components/Header.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  ChangeEvent, FormEvent, useContext, useState,
+} from 'react';
+import { GlobalContext } from '../context/GlobalContext';
+import { ErrorEnum } from '../types/ErrorEnum';
+import { USER_ID } from '../constants';
+
+export const Header = () => {
+  const [query, setQuery] = useState('');
+  const {
+    addTodo,
+    setErrorAndClear,
+    isInputDisabled,
+    setTempTodo,
+  } = useContext(GlobalContext);
+
+  const onQueryChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setQuery(e.target.value);
+  };
+
+  const onSubmitHandler = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!query.trim()) {
+      setErrorAndClear(ErrorEnum.EMPTY_TITLE, 3000);
+
+      return;
+    }
+
+    if (query.trim()) {
+      const todo = {
+        title: query,
+        userId: USER_ID,
+        completed: false,
+        id: 0,
+      };
+
+      setTempTodo(todo);
+      addTodo({
+        title: query,
+        userId: USER_ID,
+        completed: false,
+      });
+    }
+
+    setQuery('');
+  };
+
+  return (
+    <header className="todoapp__header">
+      <button
+        type="button"
+        className="todoapp__toggle-all active"
+        aria-label="Toggle All"
+      />
```

Comment:
no see any condition for show or hide toggle all button
![Screenshot from 2023-08-26 17-37-14](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/a8ad45d7-71ef-4cab-89b6-ad0836a18a00)
please fix in next todo app task